src/components/visualizations/chart/highcharts/commonConfiguration.ts

```diff
@@ -109,6 +111,20 @@ const BASE_TEMPLATE: any = {
             },
         },
     },
+    xAxis: [
+        {
+            events: {
+                afterSetAxisTranslation() {
```

Comment:
Having this in base template would affect all charts with x axis. Please check:
* Does this work well for categories (i.e. non-numeric) axes in column chart (I mean if it does not break those charts)?
* Is it sufficient for bubble/scatter having this only for x axis? Is something like this not needed for y axis if numeric?

Reply:
Problem: 
<img width="689" alt="min-max-out-of-range" src="https://user-images.githubusercontent.com/28858842/60274566-7c02b200-9922-11e9-9f82-8ef96d02dde4.png">

I checked:
- With categories Axes, it have no min/max config. This event won't affect categories axes by this [line](https://github.com/gooddata/gooddata-react-components/pull/1060/files#diff-bbc27db646c31261349ee19c14d48d2eR120) 
- For yAxis (vertical axes): luckily, it have special [canvas coordinates offset](https://github.com/highcharts/highcharts/blob/v7.1.1/js/parts/Axis.js#L3602) while translate from axis values to  pixel positions. Then the pixel position which was calculated by [expression](https://github.com/highcharts/highcharts/blob/v7.1.1/js/parts/Axis.js#L3629) always pass the condition that data [point is insided](https://github.com/highcharts/highcharts/blob/v7.1.1/js/parts/Series.js#L3885) or not. Then problem won't happen with yAxis.